backend/services/issue/addIssue.js

```diff
@@ -0,0 +1,69 @@
+const { sequelize, issue, comment, user_issue, issue_label } = require('../../sequelize/models');
+
+module.exports = async (req, res) => {
+  const { uid, title, content, assigneeIds, labelIds, milestoneId } = req.body;
+  const transaction = await sequelize.transaction();
+  try {
+    // 이슈등록
+    const creaatedIssue = await issue.create(
+      {
+        title,
+        isClosed: 0,
+        milestoneId,
+      },
+      { transaction },
+    );
+
+    //   라벨 등록
+    const labelDatas = [];
+    labelIds.forEach((id) => {
+      const obj = {
+        issueId: creaatedIssue.id,
+        labelId: id,
+      };
+
+      labelDatas.push(obj);
+    });
```

Comment:
map을 사용하면 한번에 될 것 같습니다
```suggestion
    const labelDatas = labelIds.map((id) => {
      return {
        issueId: creaatedIssue.id,
        labelId: id,
      };
    });
```